qtpy/tests/test_patch_qheaderview.py

```diff
@@ -17,11 +17,12 @@ def get_qapp(icon_path=None):
 
 
 @pytest.mark.skipif(
-    QT_VERSION.startswith('5.15') or PYSIDE6 or PYQT6 or
-    ((PYSIDE2) and sys.version_info.major == 3 and sys.version_info.minor >= 8
-     and (sys.platform == 'darwin' or sys.platform.startswith('linux'))
-    ),
-    reason="Segfaults with Qt 5.15; and PySide2/Python 3.8+ on Mac and Linux")
+    QT_VERSION.startswith('5.15') or PYSIDE6 or PYQT6,
+    reason="Segfaults with Qt >=5.15")
+@pytest.mark.skipif(
+    (PYSIDE2) and sys.version_info.major == 3 and sys.version_info.minor >= 7
+     and (sys.platform == 'darwin' or sys.platform.startswith('linux')),
+    reason="PySide2/Python 3.7+ on Mac and Linux")
```

Comment:
```suggestion
    PYSIDE2 and sys.version_info.major == 3 and sys.version_info.minor >= 7
     and (sys.platform == 'darwin' or sys.platform.startswith('linux')),
    reason="Segfaults with PySide2 in Python 3.7+ on Mac and Linux")
```

More descriptive and clear